### DISCUSSION TITLE
No code helper

### DISCUSSION THREAD

**Discussion by DannyHatcher-Git** [RafaelGB/obsidian-db-folder#490] [General]
In a recent Obsidian webinar, it was mentioned that maintainers and no-code individuals can help with plugin development specifically for templater.

I was wondering if there was anything I could do to help with the development of this plugin as something that is at the very beginning of learning how to code/program.

**Comment by imed-ghomari** [comment]
There is something, I was actually helping with the documentation, but I couldn't add any videos/photos to it because I don't have access to the new theme. If you want, you could add them to the docs.

Otherwise, I've seen Rafael accept contributions in issues with the `visual` tag. So if you know CSS, maybe you could help with that.

**Reply by RafaelGB** [reply]
Thanks for opening the thread and helping with the project 😊

A pending issue is to set up a folder with examples of use.  They would be of great help

**Reply by DannyHatcher-Git** [reply]
ok that sounds great. I know a little CSS so will have a look.
I make YouTube videos so I should be able to help with that.

**Reply by RafaelGB** [reply]
with the videos is what you help the most ❤️

**Comment by imed-ghomari** [comment]
Also, I'm not sure about this, but some issues in the roadmap have story points (numbers) attributed to them. I think it's the difficulty of the task. You could look at the issues that have the lowest story points and have a go at them, maybe they're at your level. Rafael can confirm this.

**Reply by imed-ghomari** [reply]
@DannyHatcher-Git https://github.com/users/RafaelGB/projects/7/views/2?filterQuery=status%3A%22%F0%9F%A5%B6+backlog%22+story-points%3A%3C10&sortedBy%5Bdirection%5D=asc&sortedBy%5BcolumnId%5D=14566151&visibleFields=%5B%22Title%22%2C14566151%2C%22Labels%22%5D

**Reply by RafaelGB** [reply]
The story point represents the difficulty of the issue following a fibonacci serie 😄
